cmd/collector/app/collector.go

```diff
@@ -94,6 +95,7 @@ func (c *Collector) Start(options *flags.CollectorOptions) error {
 		CollectorOpts:  options,
 		Logger:         c.logger,
 		MetricsFactory: c.metricsFactory,
+		TenancyMgr:     tenancy.NewTenancyManager(&options.GRPC.Tenancy),
```

Comment:
when collector is instantiated from all-in-one, will this cause multiple tenancy managers to exist?